ops/terraform/services/static-site/README.md

```diff
@@ -0,0 +1,18 @@
+# `static-site` Terraservice
+
+This Terraservice contains the Terraform IaC for BFD's static-site documentation infrastructure. 
+
+## Usage
+
+The `base` and `common` Terraservices _must_ be applied, to the _same Terraform workspace_, prior to `apply`ing this Terraservice. This Terraservice relies on the resources created by those Terraservices, and will fail to apply if they do not exist.
```

Comment:
I think this only requires that `base` is applied for the like-named environment.